src/app/client/src/app/modules/dashboard/components/course-progress/course-progress.component.html

```diff
@@ -269,7 +269,7 @@
         </div> -->
         <div class="sb-pagination-container flex-ai-center">
           <div *ngIf="dashboarData && dashboarData.data.length >0 && !showLoader">showing {{pager.startIndex+1}} -
-            {{pager.endIndex+1}} of {{dashboarData.data.count}} users</div>
+            {{pager.endIndex+1}} {{dashboarData.data.count}} users</div>
```

Comment:
What is this value: {{pager.endIndex+1}}

Reply:
this is an existing code, calculated to show the counts of rows of the data table shown, the issue raised for the message , for example, instead of "showing 1 - 200 of users" , the message will be "showing 1 - 200 users".